src/flags.rs

```diff
@@ -3,6 +3,7 @@ use clap::{ArgMatches, Error, ErrorKind};
 #[derive(Clone, Debug, Copy)]
 pub struct Flags {
     pub display_all: bool,
+    pub display_almost_all: bool,
```

Comment:
As it should have only one of the option display_all/display_almost_all/display_only_visible active I think it would be a good idea to refactor a bit and create a "Display" enum or something like this. 

This can be made in a future PR.

What do you think of it?

Reply:
I  guess that will be a good addition, will reduce some confusion in the check in `mod.rs` line 84.

@hemreari , could you make that change too?

Reply:
> @hemreari , could you make that change too?

of course